FirebaseMessaging/Tests/UnitTests/FIRMessagingRmqManagerTest.m

```diff
@@ -81,6 +81,19 @@ - (void)testSavingSyncMessage {
   XCTAssertFalse(persistentMessage.mcsReceived);
 }
 
+- (void)testQuerySyncMessageWithRmqID {
+  // This is to make sure there is no sql injection vulnerability.
+  NSString *rmqID = @"' --";
```

Comment:
I don't understand this test case. How does this ensure there is no vulnerability?